src/web/twig/Extension.php

```diff
@@ -211,6 +211,7 @@ public function getFilters(): array
             new \Twig_SimpleFilter('index', [ArrayHelper::class, 'index']),
             new \Twig_SimpleFilter('indexOf', [$this, 'indexOfFilter']),
             new \Twig_SimpleFilter('intersect', 'array_intersect'),
+            new \Twig_SimpleFilter('json_decode', [$this, 'jsonDecodeFilter']),
```

Comment:
Pretty sure that this would work just as well –
```suggestion
            new \Twig_SimpleFilter('json_decode', 'json_decode'),
```

Reply:
Ah I had missed that `$assoc = true`.

Maybe this then?

```php
new \Twig_SimpleFilter('json_decode', [Json::class, 'decode']),
```

(and import `craft\helpers\Json` at the top)

Reply:
Good call, that sounds perfect! Ironically, I just discovered `Json::decode` last night.

I'll try to make that change within the next 24 hours and resubmit. 👍